search/src/Components/error/details.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {jsonSchemaTransform} from './utils';
+
+
+const JsonSchemaDetails = (props) => {
+    const {details} = props
+    const tfError = jsonSchemaTransform(details)
+    const tfDisplay = Object.keys(tfError).map((path) => {
+        return (
+            <div>
+                <p>{tfError[path].instances.length} {tfError[path].instances.length > 1 ? "errors": "error"} in the {path}</p>
+                {tfError[path].instances.map((error) => <p>{error.message} @ {error.path}</p>)}
```

Comment:
```suggestion
                <ol>
                {tfError[path].instances.map(error =>
                  <li key={error.path}>In {error.path}: {error.message}</li>
                )}
                </ol>
```
There might be a material-style component better than `<li>`